### PR TITLE
FUSETOOLS-2250: Create .vscode with config files automatically after Camel Quarkus project is created

### PR DESCRIPTION
**Pull request by djelinek**
# Pull Request informations

## Rebase & Merge default requirements

1. Green build for main branch
2. Wait 24 hours after PR creation
3. Green job for PR
4. Approved PR

## PR labels default process

- READY_FOR_REVIEW  &rightarrow;  REVIEW_DONE  &rightarrow;  READY_FOR_MERGE

## Tests

- [ ] Are there **Unit tests**?
- [ ] Are there **Integration tests**?
- [ ] Do we need a new **UI test**?

## PR workflow progress

1. [ ] Tagged with relevant **PR labels**
2. [ ] Green **job for PR**
3. [ ] PR was created more than **24 hours ago** or **All committers approved** it
4. [ ] Green **main** branch build

### CODE REVIEW

**Comment by djelinek** [comment]
using same configuration as for tasks completion in DAP extension (https://github.com/camel-tooling/camel-dap-client-vscode/pull/530) for `resources/*` files used for auto generation inside `.vscode` fodler

File: resources/tasks.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            "label": "Build a Camel Quarkus application as a Native executable",
+            "detail": "This task will build Camel Quarkus application with JMX and Camel Debugger enabled using GrealVM",
+            "type": "shell",
+            "command": "./mvnw",
+            "args": [
+                "install",
+                "-Dnative",
+                "-Dquarkus.native.monitoring=jmxserver,jmxclient",
+                "-Dquarkus.camel.debug.enabled=true",
+                "-Pcamel.debug" // This depends on your project
```
Comment (djelinek) [inline resources/tasks.json:14]: better to check profile naming after Camel 4.2 is released

File: resources/launch.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Run Camel application and attach Camel debugger",
```
Comment (djelinek) [inline resources/launch.json:5]: maybe I should mention `native` also here?

**Comment by djelinek** [comment]
mentioned windows issue is fixed, problem was inside UI tests dynamic waits. Implementation of files generation works fine.

File: resources/launch.json (hunk 2 above)
Comment (djelinek) [reply resources/launch.json:5]: WDYT @apupier ?

File: resources/tasks.json (hunk 1 above)
Comment (djelinek) [reply resources/tasks.json:14]: tested, profile is there in this format

File: resources/launch.json (hunk 2 above)
Comment (apupier) [reply resources/launch.json:5]: agree

**Review by apupier — APPROVED** [review]

**Comment by djelinek** [comment]
also added note into changelog

File: Changelog.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 - Update default Camel Catalog version from 4.1.0 to 4.2.0
 - Update Kamelet Catalog from 4.0.1 to 4.2.0
+- Automatically generates `.vscode` folder with `tasks.json` and `launch.json` files  after Camel Quarkus project is created. It contains predefined configurations for a build and start of a Camel Quarkus native application.
```
Comment (djelinek) [inline Changelog.md:7]: @apupier do you think it's understandable enough? I was trying to be most precise with minimum words

**Pull request merged**